xbmc/cores/AudioEngine/Sinks/AESinkAUDIOTRACK.cpp

```diff
@@ -590,15 +599,20 @@ void CAESinkAUDIOTRACK::Deinitialize()
 
   m_duration_written = 0;
   m_headPos = 0;
-  m_timestampPos = 0;
-  m_stampTimer.SetExpired();
-
+  m_offset = -1;
+  m_extTimer.SetExpired();
   m_linearmovingaverage.clear();
 
   delete m_at_jni;
   m_at_jni = NULL;
-  m_delay = 0.0;
-  m_hw_delay = 0.0;
+  uint64_t gone = CurrentHostCounter() - before;
+  uint64_t delta_ms = 1000 * gone / CurrentHostFrequency();
+  int64_t diff = m_audiotrackbuffer_sec * 1000 - delta_ms;
+  if (diff > 0)
+  {
+    CLog::Log(LOGDEBUG, "Flushing might not be properly implemented, sleeping: %d ms", diff);
```

Comment:
Please update to the current log standards using `{}`.

For more information please see the [libfmt documentation](https://fmt.dev/latest/syntax.html)